bodybank/character_generator/models.py

```diff
@@ -11,7 +12,8 @@ class Ego(models.Model):
     wil = models.IntegerField("Willpower (WIL)")
     morph = models.ForeignKey("Morph",null=True,on_delete=models.CASCADE)
     items = models.ManyToManyField("Item")
-    
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
cascade means here that if we delete a user all their data will be deleted too, is that what you want? alternative you can use SET_NULL